README.md

```diff
@@ -153,6 +153,7 @@ The app should then be available at http://localhost:5173/
 
 #### iOS and Android Simulators (with Hot Reload)
 
+Make sure to install Corvoda's required dependencies for [iOS](https://cordova.apache.org/docs/en/11.x/guide/platforms/ios/index.html)/[Android](https://cordova.apache.org/docs/en/11.x/guide/platforms/android/) before running the following commands.
```

Comment:
Is there a reason for linking to the v11 docs rather than v12 or whatever version the app currently uses? I'm not familiar with it, but I can't see any reference to a version in the codebase except [10.1.1](https://github.com/salemlf/hakubun/blob/0bbe5d55dd5cc277624d77dc13fd65829c423aec/android/variables.gradle#L15)

Also, you can drop the `index.html` on the iOS link for consistency 😊

Reply:
That's a good point, it should really link to the 10.x version instead since that's what's referenced in the variables file you mentioned. Dropping the "index.html" ending makes sense, yeah!